src/appwindow.cpp

```diff
@@ -632,6 +632,10 @@ void AppWindow::on_actionBuildInfo_triggered()
                            .arg("Windows")
 #elif defined(Q_OS_MACOS)
                            .arg("macOS")
+#elif defined(Q_OS_FREEBSD)
+                           .arg("FreeBSD")
+#elif defined(Q_OS_UNIX)
```

Comment:
UNIX isn't actually an OS, but I'll give it a pass since there isn't anything else we can put there.